gravitee-apim-console-webui/src/services-ngx/api-score.service.spec.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TestBed } from '@angular/core/testing';
+
+import { ApiScoreService } from './api-score.service';
+
+xdescribe('ApiScoreService', () => {
```

Comment:
Shouldn't be?

```suggestion
describe('ApiScoreService', () => {
```

Reply:
this is empty, placeholder service, test skipped.